components/Footer/index.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import Svg from "../Svg";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export default function Footer() {
+  const { pathname } = useRouter();
+
+  return (
+    <StyledFooter>
+      <NavLink href={"/faves"} aria-label="Navigate to favorites page">
+        <Svg
+          variant="faves"
+          color={
+            pathname === "/"
+              ? "var(--fifth-color"
+              : pathname !== "/" || pathname !== "/faves"
+              ? "var(--fifth-color"
+              : "var(--primary-color"
+          }
+        />
+      </NavLink>
+
+      <NavLink href={"/"} aria-label="Navigate to landing page">
+        <Svg
+          variant="home"
+          color={
+            pathname === "/"
+              ? "var(--primary-color)"
+              : pathname !== "/" || pathname !== "/faves"
+              ? "var(--fifth-color)"
+              : "var(--fifth-color)"
+          }
+        />
```

Comment:
This is very complicated to read. In the end, you want to achieve the following: When the current path is "/", the color of the home icon should change. This could be done so:

```suggestion
           <NavLink href={"/faves"} aria-label="Navigate to favorites page">
        <Svg variant="faves" color={"var(--fifth-color"} />
      </NavLink>

      <NavLink href={"/"} aria-label="Navigate to landing page">
        <Svg
          variant="home"
          color={pathname === "/" ? "var(--primary-color)" : "var(--fifth-color)"}
        />
      </NavLink>
```

Anyhow, I don't think it is really necessary to hide the link to the home page. If you still want to do so, I advise to use conditional rendering. Right now, you are just hiding the link by giving it the same color as the background. For accessibility reasons, I suggest to render the link conditionally. So that it is only rendered, when the current path is "/".